xtevent/xtevent.sthlp

```diff
@@ -206,9 +208,17 @@ option requires {help reghdfe} and {help ftools} to be installed. For IV estimat
 Note that standard errors may be different and singleton clusters may be dropped using {help reghdfe}. See Correia (2017).
 
 {phang}
-{opt addabsorb(varlist)} specifies additional fixed effects to be absorbed when using {help reghdfe}. By default, {cmd xtevent} includes time and
+{opt addabsorb(varlist)} specifies additional fixed effects to be absorbed when using {help reghdfe}. By default, {cmd:xtevent} includes time and
 unit fixed effects. {opt addabsorb} requires {opt reghdfe}.
 
+{phang}
+{opt cohort(varname)} specifies the variable that identifies the cohort for each unit. {opt cohort} and {opt control_cohort} indicates {cmd:xtevent} to estimate the even-time coefficients with the Interaction Weighted Estimator

```

Comment:
@Constantino-Carreto-Romero fix typos, emphasize that these options are required for SA.

@jorge-emilio-melendez Can you help to check the writing here?

Reply:
@jorpppp and @Constantino-Carreto-Romero I found a typo in line 219:

It should be "... to estimate the **event-time** coefficients with the Interaction Weighted Estimator." 
instead of **even-time**

Reply:
Thanks @jorge-emilio-melendez. @Constantino-Carreto-Romero , can you fix this, please?